src/store/index.js

```diff
@@ -1,17 +1,43 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from "vue";
+import Vuex from "vuex";
+import axios from "axios";
 
-Vue.use(Vuex)
+Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: {
+    movies: [],
+    searchQuery: "",
```

Comment:
Why do you create this variable in state if you use one from the component data?

Reply:
Good question :)
I have been change the logic of my component and linked searchQuery from local data in component to state. I quess it will be helpful in future.